db/db_impl/db_impl_secondary.cc

```diff
@@ -455,6 +458,19 @@ Status DBImplSecondary::TryCatchUpWithPrimary() {
   InstrumentedMutexLock lock_guard(&mutex_);
   s = static_cast<ReactiveVersionSet*>(versions_.get())
           ->ReadAndApply(&mutex_, &manifest_reader_, &cfds_changed);
+
+  ROCKS_LOG_INFO(immutable_db_options_.info_log, "Last sequence is %" PRIu64,
```

Comment:
How about add more information to the log statement so that readers will know it's from `TryCatchUpWithPrimary`?

Reply:
Sure.

Reply:
Logging itself contains file name and line number already so that people can know it is from TryCatchUpWithPrimary.